subhub/requirements.txt

```diff
@@ -31,7 +31,7 @@ PyYAML==5.1.1
 requests==2.22.0
 s3transfer==0.2.1
 six==1.12.0
-stripe==2.32.1
+stripe==2.33.0
```

Comment:
Please provide a list of changes between stripe 2.32.1 and 2.33.0

Reply:
```
diff --git a/.travis.yml b/.travis.yml
index 1cc1a96..3abd3f6 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -25,7 +25,7 @@ cache:
 env:
   global:
     # If changing this number, please also change it in `tests/conftest.py`.
-    - STRIPE_MOCK_VERSION=0.60.0
+    - STRIPE_MOCK_VERSION=0.63.0
 
 before_install:
   # Unpack and start stripe-mock so that the test suite can talk to it
diff --git a/CHANGELOG.md b/CHANGELOG.md
index 3c28a48..03c900d 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.33.0 - 2019-07-30
+* [#595](https://github.com/stripe/stripe-python/pull/595) Listing `BalanceTransaction` objects now uses `/v1/balance_transactions` instead of `/v1/balance/history`
+
 ## 2.32.1 - 2019-07-08
 * [#592](https://github.com/stripe/stripe-python/pull/592) Fix argument name conflict
 
diff --git a/VERSION b/VERSION
index 7780cec..3afbaeb 100644
--- a/VERSION
+++ b/VERSION
@@ -1 +1 @@
-2.32.1
+2.33.0
diff --git a/stripe/api_resources/balance_transaction.py b/stripe/api_resources/balance_transaction.py
index 1cb224c..c1f1aef 100644
--- a/stripe/api_resources/balance_transaction.py
+++ b/stripe/api_resources/balance_transaction.py
@@ -5,7 +5,3 @@ from stripe.api_resources.abstract import ListableAPIResource
 
 class BalanceTransaction(ListableAPIResource):
     OBJECT_NAME = "balance_transaction"
-
-    @classmethod
-    def class_url(cls):
-        return "/v1/balance/history"
diff --git a/stripe/version.py b/stripe/version.py
index cfb8775..b4aed5d 100644
--- a/stripe/version.py
+++ b/stripe/version.py
@@ -1 +1 @@
-VERSION = "2.32.1"
+VERSION = "2.33.0"
diff --git a/tests/api_resources/test_balance_transaction.py b/tests/api_resources/test_balance_transaction.py
index d9417cc..a05ccc1 100644
--- a/tests/api_resources/test_balance_transaction.py
+++ b/tests/api_resources/test_balance_transaction.py
@@ -9,6 +9,13 @@ TEST_RESOURCE_ID = "txn_123"
 class TestBalanceTransaction(object):
     def test_is_listable(self, request_mock):
         resources = stripe.BalanceTransaction.list()
-        request_mock.assert_requested("get", "/v1/balance/history")
+        request_mock.assert_requested("get", "/v1/balance_transactions")
         assert isinstance(resources.data, list)
         assert isinstance(resources.data[0], stripe.BalanceTransaction)
+
+    def test_is_retrievable(self, request_mock):
+        resource = stripe.BalanceTransaction.retrieve(TEST_RESOURCE_ID)
+        request_mock.assert_requested(
+            "get", "/v1/balance_transactions/%s" % TEST_RESOURCE_ID
+        )
+        assert isinstance(resource, stripe.BalanceTransaction)
diff --git a/tests/conftest.py b/tests/conftest.py
index fb358c4..eb0f378 100644
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -16,7 +16,7 @@ from tests.stripe_mock import StripeMock
 
 
 # When changing this number, don't forget to change it in `.travis.yml` too.
-MOCK_MINIMUM_VERSION = "0.60.0"
+MOCK_MINIMUM_VERSION = "0.63.0"
 
 # Starts stripe-mock if an OpenAPI spec override is found in `openapi/`, and
 # otherwise fall back to `STRIPE_MOCK_PORT` or 12111.

```

Reply:
The large changes appear to be:

+## 2.33.0 - 2019-07-30
+* [#595](https://github.com/stripe/stripe-python/pull/595) Listing `BalanceTransaction` objects now uses `/v1/balance_transactions` instead of `/v1/balance/history`
+
 ## 2.32.1 - 2019-07-08
 * [#592](https://github.com/stripe/stripe-python/pull/592) Fix argument name conflict